google/cloud/storage/_media/common.py

```diff
@@ -17,108 +17,5 @@
 Includes custom exception types, useful constants and shared helpers.
 """
 
-import http.client
-
-_SLEEP_RETRY_ERROR_MSG = (
-    "At most one of `max_cumulative_retry` and `max_retries` " "can be specified."
-)
-
 UPLOAD_CHUNK_SIZE = 262144  # 256 * 1024
 """int: Chunks in a resumable upload must come in multiples of 256 KB."""
```

Comment:
With only this constant remaining, would it make more sense to move this to the upload file?

Reply:
IIRC UPLOAD_CHUNK_SIZE is exposed in __init__.py, so rather than tear that out or modify it to expose part of upload.py I decided to leave it in place.